### PR TITLE
bundix: 1.0.4 -> 2.0.4

### PR DESCRIPTION
**Pull request by manveru**

### CODE REVIEW

File: pkgs/development/interpreters/ruby/bundix/default.nix [hunk 1]
```diff
@@ -1,20 +1,11 @@
...
-    gemName = "thor";
-    version = "0.19.1";
-    type = "gem";
-    sha256 = "08p5gx18yrbnwc6xc0mxvsfaxzgy2y9i78xq7ds0qmdm67q39y4z";
-  };
-
-in buildRubyGem {
+buildRubyGem rec {
+  inherit ruby;
+  buildInputs = [bundler];
```
Comment (zimbatm) [inline pkgs/development/interpreters/ruby/bundix/default.nix]: Isn't `nix-prefetch-scripts` also a dependency ?

File: pkgs/development/interpreters/ruby/bundix/default.nix [hunk 2]
```diff
@@ -1,20 +1,11 @@
-{ ruby, fetchgit, buildRubyGem, bundler }:
+{ buildRubyGem, makeWrapper, ruby, coreutils, bundler }:
```
Comment (zimbatm) [inline pkgs/development/interpreters/ruby/bundix/default.nix]: makeWrapper is not used

File: pkgs/development/interpreters/ruby/bundix/default.nix [hunk 3]
```diff
@@ -1,20 +1,11 @@
...
-  version = "1.0.4";
-  gemPath = [ thor bundler ];
-  src = fetchgit {
-    url = "https://github.com/cstrahan/bundix.git";
-    rev = "6dcf1f71c61584f5c9b919ee9df7b0c554862076";
-    sha256 = "1w17bvc9srcgr4ry81ispcj35g9kxihbyknmqp8rnd4h5090b7b2";
-  };
+  version = "2.0.4";
+  sha256 = "0i7fdxi6w29yxnblpckczazb79m5x03hja8sfnabndg4yjc868qs";
+  dontPatchShebangs = true;
```
Comment (zimbatm) [inline pkgs/development/interpreters/ruby/bundix/default.nix]: Why not patch the shebangs ?

**Comment by zimbatm** [comment]
What do you think of this ? https://github.com/zimbatm/nixpkgs/commit/b98b2cdcaf0d7842707b4dd708794508d665357a

**Comment by manveru** [comment]
I'm in love with it, maybe add meta property too?

**Comment by zimbatm** [comment]
I had some mistakes in that commit, here is the fixed-up stuff: #12919

**Comment by zimbatm** [comment]
Now with metadata

**Comment by zimbatm** [comment]
There is also a release-detection script but don't know how it works yet

**Pull request closed without merging**